examples/pylab_examples/multicolored_line.py

```diff
@@ -1,53 +0,0 @@
-'''
-Color parts of a line based on its properties, e.g., slope.
-'''
-
-import numpy as np
-import matplotlib.pyplot as plt
-from matplotlib.collections import LineCollection
-from matplotlib.colors import ListedColormap, BoundaryNorm
-
-x = np.linspace(0, 3 * np.pi, 500)
-y = np.sin(x)
-z = np.cos(0.5 * (x[:-1] + x[1:]))  # first derivative
-
-# Create a colormap for red, green and blue and a norm to color
-# f' < -0.5 red, f' > 0.5 blue, and the rest green
-cmap = ListedColormap(['r', 'g', 'b'])
-norm = BoundaryNorm([-1, -0.5, 0.5, 1], cmap.N)
```

Comment:
It is a shame to loose the boundary norm example here.  Maybe do two axes, one with continuous, one with boundary norm?